src/main/java/com/mojang/brigadier/tree/LiteralCommandNode.java

```diff
@@ -39,6 +42,12 @@ public String getName() {
         return literal;
     }
 
+    public boolean isCaseInsensitive() { return isCaseInsensitive; }
+
+    public void caseInsensitive(boolean isCaseInsensitive) {
```

Comment:
Remove this method, nodes should never be able to be modified after they have been created, otherwise it is possible to violate the guarantees stipulated in `CommandDispatcher#getPath()` and `CommandDispatcher#findNode()`